### PR TITLE
Add: `yabridge-bin`

### PR DESCRIPTION
**Pull request by rain1598**

### CODE REVIEW

File: packages/yabridge-bin/yabridge-bin.pacscript [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+          sudo install -D ./* /usr/lib
+}
+
+build() {
+          true
+}
+
+install() {
+          sudo install -D ./yabridgectl /usr/bin/
+          sodo install -D ./yabridge-host.exe /usr/bin/
```
Comment (wizard-28) [inline packages/yabridge-bin/yabridge-bin.pacscript:22]:
```suggestion
          sudo install -D ./yabridge-host.exe /usr/bin/
```
Spelling mistake

File: packages/yabridge-bin/yabridge-bin.pacscript [hunk 2]
```diff
@@ -0,0 +1,27 @@
+name="yabridge-bin"
+version="3.4.0"
+url="https://github.com/robbert-vdh/yabridge/releases/download/$version/yabridge-$version.tar.gz"
+build_depends="libxcb1 libxcb1:i386"
+breaks="yabridge-lts-bin"
+description="A modern and transparent way to use Windows VST2 and VST3 plugins on Linux."
+hash="9305916c90001339180ab9751513e1210053c014da469ad2850c87b03ccd7539"
+removescript="no"
```
Comment (wizard-28) [inline packages/yabridge-bin/yabridge-bin.pacscript:8]:
```suggestion
```
Not required

File: packages/yabridge-bin/yabridge-bin.pacscript (hunk 1 above)
Comment (NarutoXY) [inline packages/yabridge-bin/yabridge-bin.pacscript:22]: It's ```sudo``` Fix the typo
Edit pacakgelist file to add ```yabridge-bin```

**Review by NarutoXY — CHANGES_REQUESTED** [review]

**Review by wizard-28 — CHANGES_REQUESTED** [review]
Also, in `install` function, you have to move the binaries etc to `$STOWDIR/$name` ( `/usr/src/pacstall/yaybridge-bin` ).

For example:
```bash
sudo install -D ./yabridgectl /usr/bin/
```

Will be:
```bash
sudo install -D ./yabridgectl $STOWDIR/$name/usr/bin/
```

**Comment by rain1598** [comment]
I think I fixed it

File: packages/yabridge-bin/yabridge-bin.pacscript [hunk 3]
```diff
@@ -9,16 +9,16 @@ optdepends=("wine: 32-bit windows emulator"
 "wine64: 64-bit windows emulator")
 maintainer="rain1598 <guacamolek@yandex.com>"
 prepare() {
-          sudo install -D ./* /usr/lib
+          sudo install -D ./* /usr/lib/yabridge/
```
Comment (wizard-28) [inline packages/yabridge-bin/yabridge-bin.pacscript]:
```suggestion
          sudo install -D ./* "$STOWDIR/usr/lib/yabridge/"
```
It needs to be prefixed with `$STOWDIR`.

Basically my moving things under that directory, you are giving pacstall ability to manage it

File: packages/yabridge-bin/yabridge-bin.pacscript [hunk 4]
```diff
@@ -9,16 +9,16 @@ optdepends=("wine: 32-bit windows emulator"
...
 
 build() {
           true
 }
 
 install() {
-          sudo install -D ./yabridgectl /usr/src/pacstall/$name/usr/bin/
-          sudo install -D ./yabridge-host.exe /usr/src/pacstall/$name/usr/bin/
+          sudo install -D ./yabridgectl /$STOWDIR/$name/usr/bin/
+          sudo install -D ./yabridge-host.exe /$STOWDIR/$name/usr/bin/
```
Comment (wizard-28) [inline packages/yabridge-bin/yabridge-bin.pacscript]:
```suggestion
          sudo install -D ./yabridgectl "$STOWDIR/$name/usr/bin/"
          sudo install -D ./yabridge-host.exe "$STOWDIR/$name/usr/bin"
```

Remove extra `/`

**Review by wizard-28 — CHANGES_REQUESTED** [review]

**Review by wizard-28 — APPROVED** [review]

**Review by NarutoXY — APPROVED** [review]

**Review by wizard-28 — APPROVED** [review]

**Pull request merged**